src/renderer/components/message/MessageList.tsx

```diff
@@ -33,7 +45,7 @@ export default function MessageList({
     chatStoreDispatch,
   ] = useChatStore()
   const messageListRef = useRef(null)
-  const lastKnownScrollHeight = useRef([null, null])
+  const lastKnownScrollHeight = useRef<number>(null)
```

Comment:
Isn't this changing it from an array with two null elements to a single null ?

Reply:
I'm not sure what the array should represent here other than an typescript error.

Reply:
Yup we initialized it as an array but set it to a single number ^^ Typical js bug that's not possible with typescript.